README.md

```diff
@@ -99,35 +100,54 @@ with objects.allow_sync():
 # Not bad. Watch this, I'm async!
 ```
 
+
+More examples
+-------------
+
+Build and run with Docker Compose:
+
+```bash
+docker compose -f examples/docker-compose.yaml build
```

Comment:
Do we really need one more docker config for the example? Can we use the development one?